libs/libopen62541/Config.in

```diff
@@ -0,0 +1,119 @@
+menu "Configuration"
+        depends on PACKAGE_libopen62541
+
+config LIBOPEN62541_UA_ENABLE_SUBSCRIPTIONS
+	bool "Enable subscriptions support"
+	default y
+
+config LIBOPEN62541_UA_ENABLE_SUBSCRIPTIONS_EVENTS
+        bool "Use events for subscriptions (EXPERIMENTAL)"
+        default n
+	depends on LIBOPEN62541_UA_ENABLE_SUBSCRIPTIONS
+
+config LIBOPEN62541_UA_ENABLE_SUBSCRIPTIONS_ALARMS_CONDITIONS
+        bool "Use alarms and conditions for subscriptions (EXPERIMENTAL)"
+        default n
+        depends on LIBOPEN62541_UA_ENABLE_SUBSCRIPTIONS_EVENTS
+
+config LIBOPEN62541_UA_ENABLE_METHODCALLS
+	bool "Enable the Method service set"
+	default y
+
+config LIBOPEN62541_UA_ENABLE_PARSING
+	bool "Enable parsing human readable formats of builtin data types"
+	default y
+
+config LIBOPEN62541_UA_ENABLE_NODEMANAGEMENT
+        bool "Enable dynamic addition and removal of nodes at runtime"
+        default y
+
+config LIBOPEN62541_UA_ENABLE_IMMUTABLE_NODES
+        bool "Enable immutable nodes"
+        default y
+
+config LIBOPEN62541_UA_ENABLE_DISCOVERY
+        bool "Enable Discovery Service (LDS)"
+        default y
+
+config LIBOPEN62541_UA_ENABLE_DISCOVERY_MULTICAST
+        bool "Enable Discovery Service with multicast support (LDS-ME)"
+        default y
+	depends on LIBOPEN62541_UA_ENABLE_DISCOVERY
+
+config LIBOPEN62541_UA_ENABLE_DISCOVERY_SEMAPHORE
+        bool "Enable Discovery Semaphore support"
+        default y
+	depends on LIBOPEN62541_UA_ENABLE_DISCOVERY
+
+choice
+        prompt "Encryption library"
+        default LIBOPEN62541_NOENCRYPTION
+
+        config LIBOPEN62541_NOENCRYPTION
+                bool "No encryption"
+
+        config LIBOPEN62541_MBDEDTLS
+                bool "Encryption support using mbed TLS"
+
+        config LIBOPEN62541_OPENSSL
+                bool "Encryption support using OpenSSL"
+endchoice
+
+config LIBOPEN62541_UA_ENABLE_ENCRYPTION_TPM2
+        bool "Enable TPM support"
+        default n
```

Comment:
```suggestion
```

ref https://github.com/openwrt/openwrt/commit/8bc72ea7be3976711dacc09f0fdab061d6e5152a, it defaults to n already